presto-docs/src/main/sphinx/release/release-346.md

```diff
@@ -0,0 +1,60 @@
+# Release 345 (XX Nov 2020)
+
+## General Changes
+
+* Add support for `RANGE BETWEEN <value> PRECEDING AND <value> FOLLOWING` window frames. ({issue}`609`)
+* Add support for {func}`extract` with `TIMEZONE_HOUR` and `TIMEZONE_MINUTE` for `time with time zone` values. ({issue}`5668`)
+* Add SQL syntax for `GRANT` and `REVOKE` on schema. This is not yet used by any connector. ({issue}`4396`)
+* Add SQL syntax for `ALTER TABLE SET AUTHORIZATION` to change the table owner. ({issue}`5717`)
+* Fix incorrect handling of negative offsets for the `time with time zone` type. ({issue}`5696`)
+* Fix incorrect result when casting `time(p)` to `timestamp(p)` for precisions higher than 6. ({issue}`5736`)
+* Fix improper table alias visibility for queries that select all fields. ({issue}`5660`)
+* Fix failure when query parameter appears in a lambda expression. ({issue}`5640`)
+* Fix failure for queries containing `DISTINCT *` and fully-qualified column names in the `ORDER BY` clause. ({issue}`5647`)
+
+## Web UI Changes
+
+* Fix "Capture Snapshot" button on the Worker page. ({issue}`5759`)
+
+## JDBC Driver Changes
+
+* Support number accessor methods like `ResultSet.getLong` or`ResultSet.getDouble`
+  on `decimal` values, as well as `char` or `varchar` that can be unambiguously interpreted as numbers. ({issue}`5509`)
+* Add `SSLVerification` JDBC connection parameter that allows to skip SSL verification on `SSLVerification=NONE` or to skip SSL 
+  host name verification on `SSLVerification=CA`. ({issue}`5610`)
+* Remove legacy `useSessionTimeZone` JDBC connection parameter. ({issue}`4521`)
+  
+## BigQuery Connector Changes
+
+* Fix issue when query could return invalid results, if some column references were pruned out during query optimisation. ({issue}`5618`)
+
+## Cassandra Connector Changes
+
+* Improve performance of `INSERT` queries with batch statement. The batch size can be configured via `cassandra.batch-size`
+  configuration property. ({issue}`5047`)
+
+## Hive Connector Changes
+
+* Support coercion between `varchar` types of different lengths. ({issue}`5484`)
```

Comment:
This is about table/partition schema mismatch, but it's not obvious from the bullet's text.

Reply:
@findepi any of the following read better?

```
* Support reading partitions with `varchar(x)` typed columns when the column evolves to `varchar(y)` type
```

```
* Fix table/partition schema mismatch failure when a `varchar` typed column changes width during schema evolution.

Reply:
To be both are better than the current wording.

Reply:
Referring to "schema evolution" implies some kind of action (which may or may not be the reason for the mismatch). How about something like this? 

```
* Fix failure when the declared length of a `varchar(n)` column in the partition schema differs from the table schema.
```

Reply:
sounds good to me.